src/content/docs/tutorial-errors/respond-outages.mdx

```diff
@@ -0,0 +1,155 @@
+---
+title: Respond to service outages
+metaDescription: Learn how to monitor your system so you can quickly identify and resolve many error occurrences fast. 
+---
+
+import allEntitiesView from 'images/apm_screenshot-crop_all_entities.webp'
+
+import apiGateway from 'images/apm_screenshot-crop_api-gateway-summary.webp'
+
+import apmErrorsInboxPage from 'images/apm_screenshot-crop_errors-inbox-page.webp'
+
+import apmErrorsAnomaly from 'images/apm_screenshot-crop_errors-anomaly.webp'
+
+Errors are bound to happen. Even with an observability tool, finding the source of an error isn't as straightforward as you might assume. Think about a flooded yard: you notice water flowing near your hose bib, but the cause of the flood is actually a crack somewhere in your water main. If you assumed that the leaking hose caused the flood, you'd end up with a fixed hose but a ruined lawn. The mistake would be costly. 
+
+Error analysis takes you to the source of the issue so you can fix the crack before your system is flooded. When a team makes a new deployment or a service fails upstream, you need to dig deeper before implementing any solutions. There's no room for assumptions in error analysis.
```

Comment:
I'm not sure about crack and flooded here, maybe terms more relatable to the product. Just a small thing to consider.